src/guide/hooks.md

```diff
@@ -20,7 +20,7 @@ The helper receives the same arguments and options (`hash`, `name` etc) as any c
 
 ### Default behavior
 
-If no parameters are passed to the mustache, the default behaviour is to do nothing and ignore the whole mustache
+If no parameters are passed to the mustache, the default behavior is to do nothing and ignore the whole mustache
```

Comment:
well, technically that is correct (British) English but both are good!

Reply:
thanks, I decided to change this one for consistency - this was the only occurrence of the British version, all other are spelled as `behavior` (like in the heading of this paragraph)